Assets/MixedRealityToolkit-SDK/Features/Teleportation/MixedRealityTeleportManager.cs

```diff
@@ -40,16 +40,6 @@ public override void Initialize()
 
         private void InitializeInternal()
         {
-            if (CameraCache.Main.transform.parent == null)
-            {
-                var cameraParent = new GameObject("Body");
-                CameraCache.Main.transform.SetParent(cameraParent.transform);
-            }
-
-            // Make sure the camera is at the scene origin.
-            CameraCache.Main.transform.parent.transform.position = Vector3.zero;
```

Comment:
Do we ensure this someplace else?

Reply:
This was the question I was asking about our assumptions of whether tracker space 0,0,0 always overlaps world space 0,0,0 when the scene starts.
https://github.com/Microsoft/MixedRealityToolkit-Unity/issues/2719#issuecomment-420449199

I found a problem where the controllers were offset improperly and fixed it. I wasn't able to find any other problems with not moving the headset, so I removed this line. We will definitely hit problems if we zero out the offset between the headset and the Playspace.

Reply:
In either case the camera itself should start at the origin. There's lots of documented cases that if it doesn't start there, things break with WMR specifically.

Reply:
Right, that's why I brought up the assumption in the other thread. We should be doing this somewhere else, but I don't know where.

It is totally reasonable for us to require that Camera to be at 0,0,0 in the scene before everything starts, I'm not disagreeing with that, but once we've offset the camera via tracking we can't arbitrarily zero it out in the TeleportManager.

If the scene starts and the physical headeset is 5 meters to the left of the center of the tracking space it should not be at world space 0,0,0 on the first frame its view renders. It should be at something like 500,0,0. (That doesn't preclude it being at 0,0,0 before tracking moves it the first time though).

Look at it the other way: If a developer doesn't want to use our teleport manager, they disable it, this code never runs. That doesn't mean we still don't need to enforce a 0,0,0 camera in the scene, so we need that code to live somewhere else. If we're enforcing it somewhere else, we don't need it here.

Reply:
Maybe in the mixed reality manager, when we setup the camera profile?